src/mocks/MockConsumer.sol

```diff
@@ -39,7 +43,9 @@ contract MockConsumer is IOracleConsumerContract {
     function trigger(CustomRequestParams memory params) public returns (bool) {
         bool success = IOracle(oracle).addRequest(
             address(this),
-            abi.encode(params)
+            abi.encode(params),
```

Comment:
params have a CustomRequestParams interface, and already includes nonce and msg.sender